quartz/plugins/transformers/ofm.ts

```diff
@@ -546,12 +558,63 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>>
                     if (matches && matches.length >= 1) {
                       parent!.children.splice(index! + 2, 1)
                       const block = matches[0].slice(1)
+                      const anchor = block.startsWith("/")
+                        ? `#block-${block.slice(1)}`
+                        : `#block-${block}`
 
                       if (!Object.keys(file.data.blocks!).includes(block)) {
                         node.properties = {
                           ...node.properties,
                           id: block,
                         }
+                        node.children = [
+                          ...node.children,
+                          {
```

Comment:
id prefer to not include an element here, having the id on the node is fine

Reply:
This is the same element we use for linking to headings.

https://github.com/jackyzha0/quartz/blob/f334e78ed674957ad465d3800917142721b16b80/quartz/plugins/transformers/gfm.ts#L32-L69

Reply:
yes i know, but its standard for headers to have clickable anchors but its quite strange for paragraph to be linkable

i feel pretty strongly about not having it here